.examples/kubernetes/chart/urlaubsverwaltung/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for urlaubsverwaltung.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: synyx/urlaubsverwaltung
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```

Comment:
Werden die resource limits hier definiert oder kommen dir von wo anders rein?

Reply:
der Chart bietet die Option an, dass man hier die Limits etc setzt - jedoch hat hier i.d.R. jeder andere Werte. In seiner eigenen values.yaml kann man sie dann entsprechend überschreiben.